sgx-types/src/secinfo.rs

```diff
@@ -0,0 +1,40 @@
+//! Section 38.11
+
+use bitflags::bitflags;
+
+#[repr(C, align(64))]
+pub struct SecInfo {
+    flags: Flags,
+    ptype: PageType,
+    reserved: [u8; 62],
+}
+
+bitflags! {
+    pub struct Flags: u8 {
+        const R = 1 << 0;
+        const W = 1 << 1;
+        const X = 1 << 2;
+        const PENDING = 1 << 2;
+        const MODIFIED = 1 << 2;
+        const PR = 1 << 2;
```

Comment:
Maybe add a comment that `PR` is restricted permissions and that this comes from 38.11.11.

Reply:
Yes, please. And actually, the `add_pages()` kernel `ioctl()` just ignores most of the flags here. So that needs to be documented too. Lots of room to improve still.